Pipfile

```diff
@@ -0,0 +1,13 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[dev-packages]
+
+[packages]
+yamllint = "*"
+"Jinja2" = "*"
+
+[requires]
+python_version = "2.7"
```

Comment:
In 55594a19277c0a2f231dc1134377313fe2ea3572, we added dual-support for Python 2.7 and 3.6. Our CI also tests for both versions. Could you also make sure Python 3.6 is included for testing?